templates/login.html

```diff
@@ -2,18 +2,60 @@
 
 {% block links %}
 <link rel="shortcut icon" href="{{url_for('static', filename='booksearch.jpg')}}" />
-<link rel ="stylesheet" href = "{{url_for('static', filename='reg.css')}}" />
+<link rel ="stylesheet" href = "{{url_for('static', filename='book_page.css')}}" />
 {% endblock %}
 {% block styles %}
-<style>
-  body {
-background-image: url('static/books.jpg'); 
-color: white; 
-background-size: cover; 
-background-repeat: no-repeat;
-}
+<style> 
 </style>
 {% endblock %}
+{% block scripts%}
+  <script type="text/javascript"> 
+  function bookdisplay(ele) {
+    console.log(ele.id)
+    var requestobj = {}
+    requestobj["isbn"] = ele.id
+    var url = "/api/book?isbn="+ele.id
+    // document.getElementById("id").innerHTML=requestobj.isbn;
```

Comment:
remove all the commented pieces of code.

Reply:
updated..